scripts/run_incast_simulation.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import os
+import subprocess
+import argparse
+import time
+
+
+def main(args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--senders", type=int, help="Number of senders in topology", default=30
+    )
+    parser.add_argument(
+        "--receivers", type=int, help="Number of receivers in topology", default=1
+    )
+    parser.add_argument("--time", type=int, help="Simulation time, ns", default=10000)
+    parser.add_argument(
+        "--packets", type=int, help="Number of sending packets", default=100
+    )
+    parsed_args = parser.parse_args()
+
+    topology_config_name = (
+        f"incast-{parsed_args.senders}-to-{parsed_args.receivers}-topology.yml"
+    )
+    simulation_config_name = (
+        f"incast_{parsed_args.senders}_to_{parsed_args.receivers}_simulation.yml"
+    )
+
+    bottleneck_args = [
+        "python3",
+        "bottleneck.py",
+        "--senders",
+        str(parsed_args.senders),
+        "--receivers",
+        str(parsed_args.receivers),
+        "--topology",
+        topology_config_name,
+        "--simulation",
+        simulation_config_name,
+        "--packets",
+        str(parsed_args.packets),
+        "--simulation-time",
+        str(parsed_args.time),
+    ]
+
+    os.chdir("configuration_examples/generator")
+    bottleneck_result = subprocess.run(bottleneck_args, capture_output=True)
+    if bottleneck_result.returncode != 0:
+        print(f"Error in bottleneck.py.")
+        print(f"Output: {bottleneck_result.stderr.decode()}")
+        os.chdir("../..")
+        exit(1)
```

Comment:
Lets move it in separate function. Also I think using `os.chdir` is some confusing, its better to use `os.path.join`. And lets replace `exit(1)` with standard solution from `subprocess`
```suggestion
    generator_dir_path = "configuration_examples/generator"
    topology_config_path = os.path.join(generator_dir_path, topology_config_name)
    simulation_config_path = os.path.join(generator_dir_path, simulation_config_name)
    bottleneck_args = [
        "python3",
        "bottleneck.py",
        "--senders",
        str(parsed_args.senders),
        "--receivers",
        str(parsed_args.receivers),
        "--topology",
        topology_config_path,
        "--simulation",
        simulation_config_path,
        "--packets",
        str(parsed_args.packets),
        "--simulation-time",
        str(parsed_args.time),
    ]

    subprocess.run(bottleneck_args, capture_output=True, check=True)
    
```